test/app/exercises_test.rb

```diff
@@ -9,11 +9,27 @@ def app
     ExercismWeb::App
   end
 
-  def test_excercises_by_key
-    alice = User.create(username: 'alice', github_id: 1)
-    exercise = UserExercise.create(user: alice, language: 'go', slug: 'one')
-    submission = Submission.create(user: alice, language: 'go', slug: 'bob', user_exercise: exercise)
+  attr_reader :alice, :exercise, :submission
+
+  def setup
+    super
+    @alice = User.create(username: 'alice', github_id: 1)
+    @exercise = UserExercise.create(user: alice, language: 'go', slug: 'one')
+    @submission = Submission.create(user: alice, language: 'go',
+                                    slug: 'bob', user_exercise: exercise)
+  end
```

Comment:
The exercise and the submission need to have the same slug. I'm not sure what happened with that old test, that had different slugs.